### DISCUSSION TITLE
removed --current-user option on version 29.2

### DISCUSSION THREAD

**Discussion by PM-92** [grahampugh/erase-install#404] [General]
Hello,

I used the --current-user function on version 27.3 so that Mac Silicon users do not have to type their login but this option no longer works on version 29.2.
Has it been renamed or completely removed?

Thanks you

**Comment by grahampugh** [comment]
The current user option is not required in 28+ because it is pre-filled into the username field. It can be changed by the user for cases where the current logged in user account cannot perform the update.
